### ISSUE TITLE
Issue with Facebook SDK Initialization in Plugin v0.0.81

### ISSUE THREAD

**Issue by boengli** [Cap-go/capacitor-social-login#88]
I encountered the following error while using the plugin version 0.0.81:

`Fatal Exception: java.lang.RuntimeException: Unable to start activity ComponentInfo{com.DaringDestiny.app/com.facebook.FacebookActivity}: A valid Facebook app id must be set in the AndroidManifest.xml or set by calling FacebookSdk.setApplicationId before initializing the sdk.`

In my app, I only use Google Sign-In and do not utilize any Facebook-related functionality. Despite this, the plugin seems to attempt initializing the Facebook SDK, which causes this issue.

Could you clarify why the plugin tries to initialize the Facebook SDK even when Facebook isn’t in use, or suggest a way to prevent this behavior?

Thank you for your help!

**Comment by WcaleNieWolny** [comment]
CC: @riderx

**Comment by riderx** [comment]
Hello @boengli this happen for a "dumb" reason.
The plugin include the Facebook lib, and the lib has a some messages to help devs understand why things not working.
So basically, they display this message automatically.
We cannot do anything about it for now, unless @WcaleNieWolny maybe make a magic trick to make the lib fail to fail to init

**Comment by WcaleNieWolny** [comment]
@riderx This issue can be resolved easily with some conditional compilation. It should not be too difficult

**Comment by WcaleNieWolny** [comment]
Ok, I give up.
This thing is beyond impossible in Java

**Comment by riderx** [comment]
i think i found a way, i published

**Comment by boengli** [comment]
At first glance, version 0.0.83 looks great—thank you! 🙌

However, I ran into an issue with 0.0.87. Whenever I try using it, I consistently get the error: “You cannot use scope without modifying the main activity.” This happens despite my current implementation working perfectly fine with 0.0.83.  🤔

**Comment by WcaleNieWolny** [comment]
The version 0.0.87 should have been released as 1.0.0.

Please follow [this](https://github.com/Cap-go/capacitor-social-login/blob/main/docs/migration_to_v1.md#migration-from-the-v0-main-version) migration guide for it

**Comment by WcaleNieWolny** [comment]
I assume that you get this error when using Google login?

**Comment by boengli** [comment]
@WcaleNieWolny
Yes, that’s right. I just reviewed your link and noticed I haven’t made the necessary modifications to MainActivity. 
That could be the issue—thanks for the insight!

**Comment by WcaleNieWolny** [comment]
@riderx we should release 1.0.0. I don't know why we did not

**Comment by riderx** [comment]
i think we forgot, let's do it

**Comment by boengli** [comment]
**BTW in your Guide (migrate v1) for modifying MainActivity are some issues (see code comments):**

```
import com.getcapacitor.BridgeActivity;
import com.getcapacitor.Plugin;  // was missing
import com.getcapacitor.PluginHandle;  // was missing
import ee.forgr.capacitor.social.login.GoogleProvider;
import ee.forgr.capacitor.social.login.SocialLoginPlugin;  // ; missing
import ee.forgr.capacitor.social.login.ModifiedMainActivityForSocialLoginPlugin;
import android.content.Intent;
import android.util.Log;  // was missing

// ModifiedMainActivityForSocialLoginPlugin is VERY VERY important !!!!!!    
public class MainActivity extends BridgeActivity implements ModifiedMainActivityForSocialLoginPlugin {} 
// } closed too early
...
```

**Comment by WcaleNieWolny** [comment]
@boengli Thanks, resolved in https://github.com/Cap-go/capacitor-social-login/commit/b3d35ad8ce8c5d006a92a78ff1cc66c99e8f57a1
